server/src/routes/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,12 @@
+class ReviewSerializer {
+  static async getSummary(review) {
+    const allowedAttributes = ["id", "comment", "rating", "trailId", "userId"]
```

Comment:
We don't need to include `userId` in the allowed attributes, we would only get this value after executing a relatedQuery()